src/app/public/modules/date-range-picker/types/date-range-calculator.ts

```diff
@@ -23,17 +23,33 @@ import {
 } from './date-range';
 
 export class SkyDateRangeCalculator {
```

Comment:
Do we need a description to define `SkyDateRangeCalculator`?

Reply:
Not really sure what to say for this one. I was thinking it was kind of self-explanatory - it just represents the calculator. Any ideas?